### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### Activity Requirements - Vet Clinic ( add "join table" for visits )

- Create a table named `vets`. ✅
- Create a "join table" called `specializations`. ✅
- Create a "join table" called `visits`. ✅
- Insert Values.
---
- Write queries to answer the following:
   - Who was the last animal seen by William Tatcher? ✅
image
   - How many different animals did Stephanie Mendez see? ✅
image
   - List all vets and their specialties, including vets with no specialties. ✅
image
   - List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020. ✅
image
   - What animal has the most visits to vets? ✅
image
   - Who was Maisy Smith's first visit? ✅
image
   - Details for most recent visit: animal information, vet information, and date of visit. ✅
image
   - How many visits were with a vet that did not specialize in that animal's species? ✅
image
   - What specialty should Maisy Smith consider getting? Look for the species she gets the most. ✅
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -141,4 +141,68 @@ LEFT JOIN animals a
...
+SELECT a.name FROM animals a RIGHT JOIN visits v 
+  ON a.id = v.animal_id 
+  WHERE v.vet_id = 1
+  ORDER BY v.visit_date DESC
+  LIMIT 1;
+/*How many different animals did Stephanie Mendez see?*/
+SELECT COUNT(DISTINCT a.name) 
+  FROM animals a RIGHT JOIN visits v 
+  ON a.id = v.animal_id 
+  WHERE v.vet_id = 2;
```
Comment (Deepakdanger) [inline queries.sql]: - [x] here we have to find How many different animals did Stephanie Mendez see
```
SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @JuanDiegoGuerra , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made 🛠

- **Fixed:** _"here we have to find How many different animals did Stephanie Mendez see"_ 🛠
- How many different animals did Stephanie Mendez see? ✅
   image


todolist1

**Review by Olamarx — APPROVED** [review]
Hi @JuanDiegoGuerra,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All queries made with join ✔️
- Data added appropriately ✔️
- Required changes fixed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**